flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/operators/schema/coordinator/SchemaRegistryRequestHandler.java

```diff
@@ -130,12 +130,13 @@ public CompletableFuture<CoordinationResponse> handleSchemaChangeRequest(
             LOG.info(
                     "Received schema change event request from table {}. Start to buffer requests for others.",
                     request.getTableId().toString());
-            if (request.getSchemaChangeEvent() instanceof CreateTableEvent
-                    && schemaManager.schemaExists(request.getTableId())) {
-                return CompletableFuture.completedFuture(
-                        wrap(new SchemaChangeResponse(Collections.emptyList())));
```

Comment:
I think this check is meant to avoid sending duplicate CreateTableEvent to downstream after restored from checkpoints. Will this patch change that behavior?